packages/autorest.typescript/CONTRIBUTING.md

```diff
@@ -0,0 +1,348 @@
+# Introduction  
+
+This package contains the High Level Client generation and the Rest Level Client generation from swagger. We will introduce both on how to contribute to high level client generation and rest level client generation.  
+
+## Typescript/Javascript High Level Client SDK generator Design
```

Comment:
As there is no diff from the original spec, could you comment the places that you have modified?

Reply:
This is the original autorest.typescript package's contributing guide. it includes how to contribute to both hlc and rlc for generation from swagger part.